### ISSUE TITLE
Documentation 

### ISSUE THREAD

**Issue by johngrabner** [PyTorchLightning/pytorch-lightning#2923]
Documentation at [https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.callbacks.lr_logger.html](url)
says:
```
>>> from pytorch_lightning import Trainer
>>> from pytorch_lightning.callbacks import LearningRateLogger
>>> lr_logger = LearningRateLogger(logging_interval='step')
>>> trainer = Trainer(callbacks=[lr_logger])
```
The above results in error
Traceback (most recent call last):
```
  File "kiss_transformer.py", line 523, in <module>
    lr_logger = LearningRateLogger(logging_interval='step')
TypeError: __init__() got an unexpected keyword argument 'logging_interval'
```

When I drop logging_interval='step'

It logs on tensorboard step 0, but I would like all steps, the same as in [logging_interval='step'](URL) example from RedEyed.

```
def configure_optimizers(self):
        optimizer = torch.optim.Adam(self.parameters(), lr=self.hparams.lr)

        def lr_foo(epoch):
            if epoch < self.hparams.warm_up_step:
                # warm up lr
                lr_scale = 0.1 ** (self.hparams.warm_up_step - epoch)
            else:
                lr_scale = 0.95 ** epoch

            return lr_scale

        scheduler = torch.optim.lr_scheduler.LambdaLR(
            optimizer,
            lr_lambda=lr_foo
        )

        return [optimizer], [scheduler]
```

**Comment by ydcjeff** [comment]
Hey @johngrabner,

this is from the recent merged PR #2847, this hasn't been released
You could try master for that if you want.

**Comment by johngrabner** [comment]
Ok, will give it a shot
